components/SendAssetForm.js

```diff
@@ -155,17 +156,18 @@ const SendAssetForm = ({
               loading={isLoading}
               variant="contained"
               disabled={
-                formattedAddresses.length < 1 ||
-                !wallet ||
-                !assetId ||
-                !csvTransactions
+                !assetBalance.success &&
+                (!wallet ||
+                  !assetId ||
+                  !csvTransactions ||
+                  0 > parseInt(assetBalance.message))
```

Comment:
The 0 > parseInt(assetBalance.message) is kind of weird. How can 0 be greater than it? Is the .message a negative value? Please explain.